src/Stratis.Bitcoin.Features.LightWallet/LightWalletSyncManager.cs

```diff
@@ -287,33 +318,58 @@ public void SyncFromHeight(int height)
                 throw new WalletException($"Invalid block height {height}. The height must be zero or higher.");
             }
 
+            // Set the syncWalletHeight to the height passed in parameter if one of the conditions is true:
+            // The syncWalletHeight is 0 meaning that the it is the first ever call to current method
+            // The height passed as parameter is equal or lower than current wallet tip
+            if (this.syncWalletHeight == 0 || height <= this.walletTip.Height)
+            {
+                // This will update the condition within the async loop below
+                this.logger.LogTrace("Setting new wallet sync height to {0}; wallet's previous sync height was {1}.", height, this.syncWalletHeight);
+                this.syncWalletHeight = height;
+            }
+            else
+            {
+                this.logger.LogTrace("Ignoring request to sync from block height {0} as the wallet is already syncing from lower block height {1}.", height, this.walletTip.Height);
+                return;
```

Comment:
Exit log

Reply:
Done

Reply:
`         this.logger.LogTrace("(-)[ALREADY_SYNCING_LOWER]"); `